### DISCUSSION TITLE
Use with firefox

### DISCUSSION THREAD

**Discussion by algebravic** [aclap-dev/video-downloadhelper#151] [Help and Q&A]
### Your question

I installed video downloader in firefox on my Intel mac.  The download icon *never* shows up at all.  I've tried it on youtube, and other sites.  How can I fix this?  Here's a site that I want to download: https://play.library.utoronto.ca/watch/fc5934fbaae6378939309797de85eb6b

**Comment by paulrouget** [comment]
Hi! Can you please share more details about your setup, like this: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by paulrouget** [comment]
And did you pin the button: https://github.com/aclap-dev/video-downloadhelper/wiki/Lost-icon
